rands.c

```diff
@@ -19,38 +20,30 @@ long _randseed = 0L;
 
 /*****************************************************/
 /**
-  \fn void RandSeed(signed long seed)
-  \brief Resets the random number seed.
+  \brief Sets the random number seed. This should only be done ONCE per iteration.

```

Comment:
`iteration` is `STEPWAT2` jargon which a `SOILWAT2` user does not understand. Provide context.